<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableEntry</class>
 <widget class="QWidget" name="VariableEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="variableSelectedCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="variableLabel">
     <property name="text">
      <string>Variable Name:   </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="variableNameLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Enter a name...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="modelLabel">
     <property name="text">
      <string>Associated Model:   </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="modelEntriesComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
